### PR TITLE
prefer-template

### PR DESCRIPTION
**Pull request by rubiesonthesky**
#### Summary

prefer template literals

#### Motivation

https://github.com/orgs/mdn/discussions/143#discussioncomment-3216375

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/idbcursor/continue/index.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ function displayData() {
     const cursor = event.target.result;
     if(cursor) {
       const listItem = document.createElement('li');
-      listItem.innerHTML = cursor.value.albumTitle + ', ' + cursor.value.year;
+      listItem.innerHTML = `${cursor.value.albumTitle}, ${cursor.value.year}`;
```
[the remark this answers is not in the document]
Comment (lionralfs) [reply files/en-us/web/api/idbcursor/continue/index.md:73]: I agree

[the remark this answers is not in the document]
Comment (lionralfs) [reply files/en-us/web/api/idbcursor/continue/index.md:73]: `innerText`?

File: files/en-us/web/api/idbcursor/delete/index.md [hunk 2]
```diff
@@ -81,7 +81,7 @@ function deleteResult() {
         };
       } else {
         const listItem = document.createElement('li');
-        listItem.innerHTML = '<strong>' + cursor.value.albumTitle + '</strong>, ' + cursor.value.year;
+        listItem.innerHTML = `<strong>${cursor.value.albumTitle}</strong>, ${cursor.value.year}`;
```
Comment (lionralfs) [inline files/en-us/web/api/idbcursor/delete/index.md:84]: Could probably remove the `strong` and do the same thing as above

File: files/en-us/web/api/media_streams_api/taking_still_photos/index.md [hunk 3]
```diff
@@ -117,7 +117,7 @@ The next task is to get the media stream:
         video.play();
     })
     .catch(function(err) {
```
Comment (lionralfs) [inline files/en-us/web/api/media_streams_api/taking_still_photos/index.md:119]: Might as well turn this into an arrow function:

```suggestion
    .catch((err) => {
```

File: files/en-us/web/api/media_streams_api/taking_still_photos/index.md [hunk 4]
```diff
@@ -358,7 +358,7 @@ If there isn't a valid image available (that is, the `width` and `height` are bo
       video.play();
     })
     .catch(function(err) {
```
Comment (lionralfs) [inline files/en-us/web/api/media_streams_api/taking_still_photos/index.md:360]:
```suggestion
    .catch((err) => {
```

File: files/en-us/web/api/mediadevices/getusermedia/index.md [hunk 5]
```diff
@@ -391,7 +391,9 @@ navigator.mediaDevices.getUserMedia(constraints)
     video.play();
   };
 })
-.catch(function(err) { console.log(err.name + ": " + err.message); }); // always check for errors at the end.
+.catch(function(err) { // always check for errors at the end.
+  console.error(`${err.name}: ${err.message}`);
+});
```
Comment (lionralfs) [inline files/en-us/web/api/mediadevices/getusermedia/index.md:396]: I'd prefer

```suggestion
.catch((err) => {
  // always check for errors at the end.
  console.error(`${err.name}: ${err.message}`);
});
```

File: files/en-us/web/api/mediadevices/getusermedia/index.md [hunk 6]
```diff
@@ -445,7 +447,7 @@ navigator.mediaDevices.getUserMedia({ audio: true, video: true })
   };
 })
 .catch(function(err) {
```
Comment (lionralfs) [inline files/en-us/web/api/mediadevices/getusermedia/index.md:449]:
```suggestion
.catch((err) => {
```

File: files/en-us/web/api/shadowroot/index.md [hunk 7]
```diff
@@ -89,11 +89,11 @@ function updateStyle(elem) {
...
-        'div {' +
-          'width: ' + elem.getAttribute('l') + 'px;' +
-          'height: ' + elem.getAttribute('l') + 'px;' +
-          'background-color: ' + elem.getAttribute('c') + ';' +
-        '}';
+        `div {` +
+          `width: ${elem.getAttribute('l')}px;` +
+          `height: ${elem.getAttribute('l')}px;` +
+          `background-color: ${elem.getAttribute('c')};` +
+        `}`;
```
Comment (lionralfs) [inline files/en-us/web/api/shadowroot/index.md:96]: Looks a little weird butt better than before

File: files/en-us/web/api/web_speech_api/using_the_web_speech_api/index.md [hunk 8]
```diff
@@ -118,17 +118,17 @@ const diagnostic = document.querySelector('.output');
 const bg = document.querySelector('html');
 const hints = document.querySelector('.hints');
 
-let colorHTML= '';
-colors.forEach(function(v, i, a){
-  console.log(v, i);
-  colorHTML += '<span style="background-color:' + v + ';"> ' + v + ' </span>';
+let colorHTML = '';
+colors.forEach(function(color, i) {
```
Comment (lionralfs) [inline files/en-us/web/api/web_speech_api/using_the_web_speech_api/index.md:122]:
```suggestion
colors.forEach((color, i) => {
```

File: files/en-us/web/api/web_speech_api/using_the_web_speech_api/index.md [hunk 9]
```diff
@@ -118,17 +118,17 @@ const diagnostic = document.querySelector('.output');
...
+  console.log(color, i);
+  colorHTML += `<span style="background-color:${color};"> ${color} </span>`;
 });
-hints.innerHTML = 'Tap/click then say a color to change the background color of the app. Try ' + colorHTML + '.';
+hints.innerHTML = `Tap or click then say a color to change the background color of the app. Try ${colorHTML}.`;
 
-document.body.onclick = function() {
+document.body.onclick = function () {
```
Comment (lionralfs) [inline files/en-us/web/api/web_speech_api/using_the_web_speech_api/index.md:128]:
```suggestion
document.body.onclick = () => {
```

File: files/en-us/web/api/mediadevices/getusermedia/index.md (hunk 6 above)
Comment (lionralfs) [reply files/en-us/web/api/mediadevices/getusermedia/index.md]: Yeah I didn't realize how many there were when I started reviewing 😅. I just thought we'd address them too, while we're at it. But I agree it's probably cleaner to follow up with a separate PR

**Review by Manuelrojo91 — COMMENTED** [review]
@manuelrios250191@gmail.com 
Duplicate of #

**Review by teoli2003 — CHANGES_REQUESTED** [review]

**Review by teoli2003 — APPROVED** [review]
👍

**Pull request merged**